api.planx.uk/gis/local_authorities/scotland.js

```diff
@@ -0,0 +1,121 @@
+import "isomorphic-fetch";
+import https from "https";
+import {
+  getQueryableConstraints,
+  makeEsriUrl,
+  setEsriGeometryType,
+  setEsriGeometry,
+  addDesignatedVariable,
+  rollupResultLayers,
+} from "../helpers.js";
+import { planningConstraints } from "./metadata/scotland";
+
+// Process local authority metadata
+const gisLayers = getQueryableConstraints(planningConstraints);
+
+// Fetch a data layer
+async function search(
+  mapServer,
+  featureName,
+  serverIndex,
+  outFields,
+  geometry,
+  geometryType
+) {
+  const { id } = planningConstraints[featureName];
+
+  let url = makeEsriUrl(mapServer, id, serverIndex, {
+    outFields,
+    geometry,
+    geometryType,
+  });
+
+  const agent = new https.Agent({
+    rejectUnauthorized: false
+  });
+  return fetch(url, { agent })
```

Comment:
Our API is returning a 500 and the following FetchError unless I pass this `agent` - are we OK with this for testing or too insecure altogether? Any other ideas here?
![Screenshot from 2023-03-29 13-25-05](https://user-images.githubusercontent.com/5132349/228579135-c73aaa62-0fe7-4524-bcac-45654e8fb8d1.png)

Reply:
yep - I'll plan to flag this with TPX when I share for feedback. added a code comment directly in this file in the meantime so we don't lose track of "why" internally